src/PointwiseFunctions/Elasticity/ConstitutiveRelations/CubicCrystal.hpp

```diff
@@ -0,0 +1,138 @@
+ * \lambda \delta^{ij}\delta^{kl} (1 - \delta^{jk}) + \mu
+ * (\delta^{ik}\delta^{jl} + \delta^{il}\delta^{jk})(1 - \delta^{ij})
+ * \f]
+ *
+ * with the three independent parameters: the _Lamé parameter_ \f$\lambda\f$,
+ * the _Shear modulus_ \f$\mu\f$ and the _Poisson ratio_ \f$\nu\f$. In the
+ * parametrization chosen in this implementation we use the experimental group
+ * parameters \f$c_{11}\f$, \f$c_{12}\f$ and \f$c_{44}\f$, related by;
+ *
+ * \f[
+ * c_{11} = \frac{1 - \nu}{\nu} \lambda = \frac{(1 - \nu)E}{(1 + \nu)(1 -
+ * 2\nu)}, \quad
+ * c_{12} = \lambda = \frac{E\nu}{(1 + \nu)(1 - 2\nu)}, \quad
+ * c_{44} = \mu
+ * \f]
+ *
+ * and inversely;
+ *
+ * \f[
+ * E = \frac{(c_{11} + 2c_{12})(c_{11} - c_{12})}{c_{11} + c_{12}}, \quad
+ * \nu = (1 + \frac{c_{11}}{c_{12}})^{-1}, \quad
+ * \mu = c_{44}
+ * \f]
+ *
+ * The stress-strain relation then reduces to
+ *
+ * \f[
+ * T^{ij} = -\delta^{ij} \left( \left( c_{11} - c_{12} - 2*c_{44} \right) S^{ij}
+ * + c_{12} \mathrm{Tr}(S) \right) - 2*c_{44} S^{ij}
+ * \f]
+ *
+ * In the isotropic homogeneous case the number of independent components
+ * reduces to two and the shear modulus is given by \f$\mu
+ * =\frac{E}{2(1+\nu)}\f$.
+ */
+
+class CubicCrystal : public ConstitutiveRelation<3> {
+ public:
+  static constexpr size_t volume_dim = 3;
+
+  struct C_11 {
+    using type = double;
+    static constexpr OptionString help = {"C12 parameter for a cubic crystal"};
+    static type lower_bound() noexcept { return 0.0; }
+  };
+
+  struct C_12 {
+    using type = double;
+    static constexpr OptionString help = {"c_12 parameter for a cubic crystal"};
+    static type lower_bound() noexcept { return 0; }
+  };
+
+  struct C_44 {
+    using type = double;
+    static constexpr OptionString help = {"c_44 parameter for a cubic crystal"};
+    static type lower_bound() noexcept { return 0.0; }
+  };
+
+  using options = tmpl::list<C_11, C_12, C_44>;
+
+  static constexpr OptionString help = {
+      "A constitutive relation that describes a cubic, crystalline material in "
+      "terms of the three independent group paremeters. The parameters "
+      "are measured in units of stress, typically Pascals."};
+
+  CubicCrystal() = default;
+  CubicCrystal(const CubicCrystal&) = default;
+  CubicCrystal& operator=(const CubicCrystal&) = default;
+  CubicCrystal(CubicCrystal&&) = default;
+  CubicCrystal& operator=(CubicCrystal&&) = default;
+  ~CubicCrystal() override = default;
+
+  CubicCrystal(double c_11, double c_12, double c_44) noexcept;
+
+  /// The constitutive relation that characterizes the elastic properties of a
+  /// material
+  tnsr::II<DataVector, 3> stress(const tnsr::ii<DataVector, 3>& strain,
+                                 const tnsr::I<DataVector, 3>& x) const
+      noexcept override;
+
+  /// The 1st group parameter \f$c_{11} = \frac{1 - \nu}{\nu} \lambda\f$
+  double c_11() const noexcept;
+  /// The 2nd group parameter; the Lamè parameter \f$c_{12} = \lambda\f$
```

Comment:
don't use non-ascii characters (Lame) (just use latex for the accent)

Reply:
\`{e} does not work for me. I looked but did not find examples in the code where accents were written with latex or just dropped.

Reply:
did you try `\f$\mathrm{Lam\`{e}}\f$`?

Reply:
I did. \f$\mathrm{Lam\acute{e}}\f$ works, but not with the underscores _Lamé parameter_.